content/webapp/components/Card/Card.tsx

```diff
@@ -63,15 +63,15 @@ export const CardBody = styled(Space).attrs(() => ({
 
   ${props =>
     props.theme.makeSpacePropertyValues('l', ['padding-bottom'], false, {
-      small: 8,
-      medium: 8,
-      large: 8,
+      small: 5,
+      medium: 5,
+      large: 5,
     })}
 
   .card-theme.card-theme--transparent & {
     padding-left: 0;
     padding-right: 0;
-    justify-content: unset;
+    flex: none;
```

Comment:
The explanation and screenshots in the PR are really nice; maybe you could toss in a comment explaining why this is important?

(Maybe it is obvious, but to a front-end novice like me I’m not sure I'd get the significance of this.)